### PR TITLE
Nearcast

### PR DESCRIPTION
**Pull request by leecronce**
Initial pull request to submit NearCast processing/viewing code to upstream.

### CODE REVIEW

File: forest/data.py [hunk 1]
```diff
@@ -427,15 +425,7 @@ def load_image_pts(path, variable, pts_3d, pts_4d):
...
         lons, lats, values, fraction)
 
-    # Roll input data into [-180, 180] range
-    if np.any(lons > 180.0):
-        shift_by = np.sum(lons > 180.0)
-        lons[lons > 180.0] -= 360.
-        lons = np.roll(lons, shift_by)
-        values = np.roll(values, shift_by, axis=1)
-
     image = geo.stretch_image(lons, lats, values)
```
[the remark this answers is not in the document]
Comment (leecronce) [reply forest/data.py]: I see that this got removed somehow.  Perhaps I fat-fingered it when deleting some print statements, but this code doesn't have any interaction with the NearCast code.  So, by all means, please correct my mistake.  As for the pygrib dependency, I can definitely add in the try/except block, but if it's not present, the NearCast code will not work.  Is there a different way to handle this?

[the remark this answers is not in the document]
Comment (andrewgryan) [reply forest/data.py]: @leecronce the try/except is only for the time being to make forest run on the server which doesn't have the library, it'll still work with your environment. We're planning on updating the deployment to give ourselves control over which libraries are deployed so it won't be an issue then

[the remark this answers is not in the document]
Comment (leecronce) [reply forest/data.py]: OK.  I can add that in.  Quick question, though.  Do I need to update my fork prior to adding in the code block, and then commit, push, and request new pull?

[the remark this answers is not in the document]
Comment (andrewgryan) [reply forest/data.py]: Nope, just a simple `git push` will update your fork and this pull request. This is effectively a mirror of your branch

[the remark this answers is not in the document]
Comment (leecronce) [reply forest/data.py]: OK, I've added the try/except block and pushed it.

[the remark this answers is not in the document]
Comment (andrewgryan) [reply forest/data.py]: Cool thanks, I'll resolve the conflicts and merge it back.

**Comment by andrewgryan** [comment]
@leecronce congratulations you're now a contributor to FOREST! Merry Christmas and have a Happy New Year!

**Pull request merged**